utils/releaser/src/ReleaseWorker/Release/MergeBranchToMasterReleaseWorker.php

```diff
@@ -33,7 +33,7 @@ public function getPriority(): int
      */
     public function work(Version $version): void
     {
-        $this->symfonyStyle->note('You need to create a merge commit, see https://github.com/shopsys/shopsys/blob/master/docs/contributing/merging-to-master-on-github.md for detailed instructions.');
+        $this->symfonyStyle->note('You need to create a merge commit, see https://github.com/shopsys/shopsys/blob/master/docs/contributing/merging-on-github.md for detailed instructions.');
```

Comment:
You forgot to update `CONTRIBUTING.md` file

Reply:
nice catch, fixed :+1: